contrib/sssd.spec.in

```diff
@@ -197,6 +197,8 @@ BuildRequires: nss_wrapper
 BuildRequires: libnl3-devel
 %if (0%{?use_systemd} == 1)
 BuildRequires: systemd-devel
+%else
+BuildRequires: initscripts
 %endif
```

Comment:
The package initscripts was installed as part of build root for me on CentOS
and IIRC epel packaging guidelines does not recommend duplicate BuildRequires